Tiltfile

```diff
@@ -28,7 +28,10 @@ def go_lint(all_go_files):
   local_resource("go_lint", "make lint", deps=all_go_files)
 
 def get_all_ts_files(path):
-  return str(local('cd %s && find . -type f -name "*.ts*" | grep -v node_modules | grep -v __snapshots__' % path)).split("\n")
+  res = str(local('cd %s && find . -type f -name "*.ts*" | grep -v node_modules | grep -v __snapshots__' % path)).split("\n")
+  # TODO(dmiller): this thing always has a spare `""` hanging out at the end. Just `pop()` it off
+  res.pop()
```

Comment:
nit, since there's a TODO: probably a little cleaner to `.rstrip()` before the `.split`, which feels like it's common enough we should build it in